_assets/css/index.scss

```
@@ -325,9 +325,16 @@ code {
   @include u-bg('accent-warm-light');
   @include u-padding-x('05');
   @include u-padding-y(1px);
+  @include u-font-size("mono", "xs");
+  @include u-display("block");
   border-radius: radius('sm');
   word-wrap: break-word;
   white-space: pre-wrap;
+  line-height: line-height("mono", 1);
```

This might benefit from a bit more padding, but otherwise looks good!  I'm not sure if this is the right spot but it seems more right than the other one too me:

```suggestion
  line-height: line-height("mono", 1);
  padding: 3px;
```